packaging/installer/kickstart-static64.sh

```diff
@@ -350,7 +350,7 @@ else
 fi
 
 if ! grep netdata-latest.gz.run "${TMPDIR}/sha256sum.txt" | safe_sha256sum -c - > /dev/null 2>&1; then
-  fatal "Static binary checksum validation failed. Stopping netdata installation and leaving binary in ${TMPDIR}"
+  fatal "Static binary checksum validation failed. Stopping netdata installation and leaving binary in ${TMPDIR}\nUsually this is a result of an older copy of the file being cached somehere and can be resolved by simply retrying in an hour."
```

Comment:
Just a few suggestions to copy-paste to the other blocks, too. Is there a more precise word than "somewhere" we could use? Personally, I can't tell whether that implies cached on my system or the platform where we host the kickstart files.

```suggestion
  fatal "Static binary checksum validation failed. Stopping Netdata Agent installation and leaving binary in ${TMPDIR}\nUsually this is a result of an older copy of the file being cached somewhere and can be resolved by retrying in an hour."
```

Reply:
‘somewhere upstream’ maybe? The problem is that we can’t be certain where. It’s most likely to be stale data on a GCS edge node, but if the user is connecting through a proxy it may be cached there instead

Reply:
"Somewhere upstream" is helpful, thanks.